ui/lib/core/addon/templates/components/box-radio.hbs

```diff
@@ -1,66 +1,68 @@
-{{#if disabled}}
+{{#if @disabled}}
 <div class="box-radio-spacing">
   <ToolTip
     @verticalPosition="above"
     @horizontalPosition="center"
     as |T|>
     <T.trigger @tabindex="-1">
       <label
-        for={{type}}
+        for={{@type}}
         class="box-radio is-disabled is-marginless"
         data-test-mount-type-radio
-        data-test-mount-type={{type}}
+        data-test-mount-type={{@type}}
+        {{on 'click' (fn this.handleClick @onRadioChange @type)}}
```

Comment:
👏 So excited we can use this now

Reply:
I think we could clean this up (and make the component file more obvious for how this works) by passing only `@type` into this.handleClick (and I'll comment the corresponding changes on the js file)